definitions/tools/pvacseq.cwl

```diff
@@ -16,7 +16,7 @@ arguments: [
     "/opt/conda/bin/pvacseq", "run",
     "--iedb-install-directory", "/opt/iedb",
     "--pass-only",
-    { position: 5, valueFrom: $(runtime.outdir) },
+    { position: 5, valueFrom: "pvacseq_epitopes" },
```

Comment:
I'm not super familiar with how Cromwell resolves cases like this. Where would it create this directory? Inside of the process's working directory?

Reply:
Yup, within the `execution` directory for this step